models/blockInput.go

```diff
@@ -49,12 +89,19 @@ func (ths *InputBlock) Validate() (bool, []error) {
 		errs = append(errs, errors.New("invalid input block type"))
 	}
 
-	if ths.Input.Type == InputBlockObjectTypeRadio && len(ths.Input.Options) == 0 {
-		errs = append(errs, errors.New("radio input must have options"))
+	if ths.Input.Name == "" {
+		errs = append(errs, errors.New("input block field 'name' cannot be empty"))
+	}
+
+	if ths.Input.Value == "" {
+		errs = append(errs, errors.New("input block field 'name' cannot be empty"))
 	}
 
-	if ths.Input.Type == InputBlockObjectTypeCounter && ths.Input.GroupID == "" {
-		errs = append(errs, errors.New("counter input must have group ID defined"))
+	if (ths.Input.Type == InputBlockObjectTypeRadio || ths.Input.Type == InputBlockObjectTypeSelect ||
```

Comment:
yg InputBlockObjectTypeDistrictSelect ini ga perlu karena optionnya itu dpt dari query ke districtOption, itu yg handle dari FE, kita nanti yg sediain query grpahql untuk dpt in district option